CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## Versions
 
 ## x.x.x
+    * Add autoRefresh to native AdView - <AppLovinMAX.AdView autoRefresh={true/false} .../>
```

Comment:
nit (how we usually reference it in our docs is "autorefresh" or "auto-refresh"), and "Native UI Component"

```suggestion
    * Add autorefresh support to banner and MREC native UI components: `<AppLovinMAX.AdView autoRefresh={true/false} .../>`
```